target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -314,10 +333,16 @@
 
 		usb30@0 {
 			status = "ok";
+
+			pinctrl-0 = <&usb0_vcc_pin>;
```

Comment:
You can make the same way of enabling usb on all other boards :) or I can make a new PR